src/types.ts

```diff
@@ -37,6 +37,14 @@ export interface Options {
    * @see https://electronjs.org/docs/api/browser-window#winloadurlurl-options
    */
   index: string | false;
+  /**
+   * The options passed when loading the index URL in the menubar's
+   * browserWindow. Everything browserWindow.loadUrl supports is supported;
+   * this object is simply passed onto browserWindow.loadUrl
+   * @default `{}`
+   * @see https://electronjs.org/docs/api/browser-window#winloadurlurl-options
+   */
+  indexLoadOptions?: object;
```

Comment:
instead of object, can we put the actual fields?

```ts
indexLoadOptions?: {
  userAgent?: string
  //etc
}
```

Reply:
Actuall, there's a `import { LoadURLOptions } from 'electron'` already available.

On the same topic, could we rename `indexLoadOptions` to `loadUrlOptions` to keep the same naming as Electron?

Reply:
OK, can do. I thought it would be hassle to keep it in sync. 

Great if we can import it but I just tried to log `LoadURLOptions` and I got `undefined`. Where did you see that? I can't find anything online

Reply:
>  just tried to log LoadURLOptions

Yeah, it's a TypeScript interface, it gets ripped off when compiling to JS.

> Where did you see that? I can't find anything online

It's generated from Electron docs by https://github.com/electron/typescript-definitions, so it's not really online. But you can search for `LoadURLOptions` in your local `node_modules/electron/electron.d.ts`

Reply:
Thanks for explaining. I'm new to TypeScript. That's handy.